src/styles/antd-overrides/button.scss

```diff
@@ -132,3 +132,7 @@
   background-color: var(--background-l2);
   box-shadow: 0px 8px 12px 2px #000000;
 }
+
+.copyIcon:hover {
```

Comment:
**suggestion**: This isn't technically an ant-d override (same with the feedback button styles above). Worth thinking about moving these to their own file so we get a clearer separation. All g for now